src/test/java/hometask3/shape/getSquare.java

```diff
@@ -0,0 +1,28 @@
+package hometask3.shape;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class getSquare {
```

Comment:
Please rename this class according to Java Naming Conventions (capitalize it) and add error messages to all asserts